doc/man/Makefile.am

```diff
@@ -0,0 +1,108 @@
+# Man pages are only built if the --enable-man-pages option was
+# passed to the configure script.
+#
+# They should always be built before creating a distribution tarball.
```

Comment:
What happens if we do make dist on a ./configure --disable-man-pages ? I would be tempted to make the make dist error out, so we don't create tarballs with missing doc by mistake.